src/Table.php

```diff
@@ -509,6 +509,14 @@ public function onRowClick($action): void
         $this->addClass('selectable');
         $this->js(true)->find('tbody')->css('cursor', 'pointer');
 
+        /* Do not trigger row click event if click stems from row content
+         like checkboxes, or links, marked as atk4-norowclick
+         */
+        $this->js(true)->find('.atk4-norowclick')->on(
```

Comment:
after test is added, before merge, consider to just exclude `a` and `checkbox` tags instead

Reply:
I do not think it is a good idea: The developer might be willing to add clickable labels, or other items into the columns, that are clickable - by adding atk4-norowclick he can easily ensure the correct behaviour for those as well.

Reply:
yes, but is there a case when the exclusion is not wanted for `a`? If not, we only generate too complex component code as the tests shown...

Reply:
A label can sometimes be configured to have an on('click') event to trigger something.

Reply:
maybe `a, checkbox, .atk4-norowclick`